src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -324,6 +336,19 @@ LibraryServiceApi.prototype.createShelf = function createShelf() {
  * @returns {gax.EventEmitter} - the event emitter to handle the call
  *   status.
  * @throws an error if the RPC is aborted.
+ *
+ * @example
+ *
+ * var api = libraryV1.libraryServiceApi();
+ * var formattedName = LibraryServiceApi.shelfPath("[SHELF]");
```

Comment:
Oh, I missed this...  the path template functions now blong to the builder function. So it's like `libraryV1.libraryServiceApi.shelfPath("[SHELF]");`